fjord/analytics/tests/test_views.py

```diff
@@ -58,6 +58,9 @@ def test_front_page(self):
         eq_(pq('.block.count strong').text(), '7')
         eq_(len(pq('li.opinion')), 7)
 
+        assert '&amp;raquo;' not in r.content
+        assert '&raquo;' in r.content
```

Comment:
Mmm... I'd put this in a separate test since it's making sure our gettext is set up right. Putting it here with no comments about it will be confusing when it fails.